addons/sale_timesheet/wizard/sale_make_invoice_advance_views.xml

```diff
@@ -15,11 +15,14 @@
                     name="timesheet_invoice_date_range"
                     attrs="{'invisible': ['|', ('invoicing_timesheet_enabled', '=', False), ('advance_payment_method', '!=', 'delivered')]}"
                 >
+                    <label for="date_start_invoice_timesheet" string="Timesheets Period" attrs="{'invisible': [ '|', ('invoicing_timesheet_enabled', '=', False), ('advance_payment_method', '!=', 'delivered')]}"/>
```

Comment:
We need to update the pot file otherwise  translation is break.

Reply:
yes, you are correct